### PR TITLE
Feature/aatf weee non obligated

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**

### CODE REVIEW

File: src/EA.Weee.RequestHandlers/Admin/AatfReports/GetNonObligatedWeeeReceivedAtAatfsDataCsvHandler.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            csvWriter.DefineColumn(@"Area", i => i.Area);
+            csvWriter.DefineColumn(@"Year", i => i.Year);
+            csvWriter.DefineColumn(@"Quarter", i => i.Quarter);
+            csvWriter.DefineColumn(@"Submitted by", i => i.SubmittedBy);
+            csvWriter.DefineColumn(@"Date submitted", i => i.SubmittedDate);
+            csvWriter.DefineColumn(@"Organisation name", i => i.OrganisationName);
+            csvWriter.DefineColumn(@"Category", i => i.Category);
+            csvWriter.DefineColumn(@"Total non-obligated WEEE received (t)", i => i.TotalNonObligatedWeeeReceived);
+            csvWriter.DefineColumn(@"Non-obligated WEEE kept / retained by DCFs (t)", i => i.TotalNonObligatedWeeeReceivedFromDcf);
+            var fileContent = csvWriter.Write(items);
```
Comment (SumeN) [inline src/EA.Weee.RequestHandlers/Admin/AatfReports/GetNonObligatedWeeeReceivedAtAatfsDataCsvHandler.cs]: Column names needs to be updated like Date submitted (GMT), WROS Pan area team

Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/Admin/AatfReports/GetNonObligatedWeeeReceivedAtAatfsDataCsvHandler.cs]: Good spot...

Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/Admin/AatfReports/GetNonObligatedWeeeReceivedAtAatfsDataCsvHandler.cs]: > Column names needs to be updated like Date submitted (GMT), WROS Pan area team

Good spot...

Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/Admin/AatfReports/GetNonObligatedWeeeReceivedAtAatfsDataCsvHandler.cs]: Updated as removed the authority, area and wros area parameters

**Pull request merged**